.github/workflows/go.yml

```diff
@@ -68,10 +72,9 @@ jobs:
             bindata_linux
             pkg/assets/zz_generated_offsets_linux.go
             embedded-bins/Makefile.variables
-
           key: ${{ runner.os }}-embedded-bins-${{ hashFiles('**/embedded-bins/**/*') }}
           restore-keys: |
-            ${{ runner.os }}-embedded-bins-${{ hashFiles('**/embedded-bins/**/*') }}
+            ${{ runner.os }}-embedded-bins-
```

Comment:
Removing the exact restore key will bring in the latest cache, `make` should have wits to see what needs to be rebuilt.